include/seqan/score/score_matrix_data.h

```diff
@@ -307,21 +307,21 @@ typedef Blosum62_ ScoreSpecBlosum62;
 typedef Score<int, ScoreMatrix<AminoAcid, ScoreSpecBlosum62> > Blosum62;
 
 
-template <>
-struct ScoringMatrixData_<int, AminoAcid, ScoreSpecBlosum62> {
+template <typename TValue>
+struct ScoringMatrixData_<TValue, AminoAcid, ScoreSpecBlosum62> {
     enum {
         VALUE_SIZE = ValueSize<AminoAcid>::VALUE,
         TAB_SIZE = VALUE_SIZE * VALUE_SIZE
     };
 
-    static inline int const * getData() {
+    static inline TValue const * getData() {
         // Matrix made by matblas from blosum62.iij
         // * column uses minimum score
         // BLOSUM Clustered Scoring Matrix in 1/2 Bit Units
         // Blocks Database = /data/blocks_5.0/blocks.dat
         // Cluster Percentage: >= 62
         // Entropy =   0.6979, Expected =  -0.5209
-        static int const _data[TAB_SIZE] = {
+        static TValue const _data[TAB_SIZE] = {
```

Comment:
is this in a separate commit marking it as a fix?

Reply:
Probably 👻